Makefile

```diff
@@ -8,7 +8,7 @@ help: ## Describe available tasks
 
 .DEFAULT_GOAL := help
 
-target/json-ld-0.1.0.jar: deps.edn src/deps.cljs $(SOURCES)
+target/fluree-json-ld.jar: deps.edn src/deps.cljs $(SOURCES)
```

Comment:
I went back and forth about asking for this change in the other pull request. While it's nice to encode the version in the jar file name, it would get annoying to have to keep updating the make task for each version bump.

I'm glad you changed it back